mesonbuild/compilers/cpp.py

```diff
@@ -251,10 +251,13 @@ def __init__(self, exelist, version, compiler_type, is_cross, exe_wrapper=None,
     # It does not support c++/gnu++ 17 and 1z, but still does support 0x, 1y, and gnu++98.
     def get_options(self):
         opts = CPPCompiler.get_options(self)
-        opts['cpp_std'] = coredata.UserComboOption('cpp_std', 'C++ language standard to use',
+        opts.update({'cpp_std': coredata.UserComboOption('cpp_std', 'C++ language standard to use',
                                                    ['none', 'c++98', 'c++03', 'c++0x', 'c++11', 'c++14', 'c++1y',
                                                     'gnu++98', 'gnu++03', 'gnu++0x', 'gnu++11', 'gnu++14', 'gnu++1y'],
-                                                   'none')
+                                                   'none'),
```

Comment:
__[Flake8]__ [E126] continuation line over-indented for hanging indent ([link](https://sider.review/gh/repos/19784232/pulls/5284#issue-3995983))

<sub>posted by Sider</sub>